Tests/TokamakStaticHTMLTests/RenderingTests.swift

```diff
@@ -205,21 +205,41 @@ final class RenderingTests: XCTestCase {
       matching: HStack {
         Rectangle()
           .frame(width: 50, height: 50)
-          .foregroundStyle(PrimaryContentStyle())
+          .foregroundStyle(HierarchicalShapeStyle.primary)
         Rectangle()
           .frame(width: 50, height: 50)
-          .foregroundStyle(SecondaryContentStyle())
+          .foregroundStyle(HierarchicalShapeStyle.secondary)
         Rectangle()
           .frame(width: 50, height: 50)
-          .foregroundStyle(TertiaryContentStyle())
+          .foregroundStyle(HierarchicalShapeStyle.tertiary)
         Rectangle()
           .frame(width: 50, height: 50)
-          .foregroundStyle(QuaternaryContentStyle())
+          .foregroundStyle(HierarchicalShapeStyle.quaternary)
       }
       .foregroundColor(.blue),
       as: .image(size: .init(width: 275, height: 100)),
       timeout: defaultSnapshotTimeout
     )
+
+    assertSnapshot(
```

Comment:
nit: time to split `RenderingTests` into separate `XCTestCase` subclasses and files I guess

Reply:
I tried to split the rendering tests into logical groupings, but let me know if you have any suggestions.